lib/src/event.dart

```diff
@@ -1035,10 +1030,7 @@ class Event extends MatrixEvent {
   /// WARNING: This does **not** test if there are only emotes. Use `event.onlyEmotes` for that!
   int get numberEmotes {
```

Comment:
this is now failing in tests and that seems to be a bug in the regex, I think?

Reply:
Where do you see it failing 👀  ?

Reply:
Should work now, please restart tests.